### DISCUSSION TITLE
Assistance Needed with Username Change After Vercel Account Migration

### DISCUSSION THREAD

**Discussion by dogangok** [vercel/community#7201] [Account Issue] [answered]
### Question

Vercel recently transitioned my personal account projects to a new free team account. When I attempted to update my username under "This is your URL namespace within Vercel," I received an error message stating, "The username is already in use."

I also get the same error, "The username is already in use," when trying to enter a new username. I’m confident that the username is still associated with my account but got mixed up during the migration Vercel performed.

Could anyone please assist me in changing my username back or provide guidance on how to resolve this issue?

Thanks in advance for your help!

**Comment by amyegan** [comment]
Hey @dogangok. Your account username should not have changed from what it was before a new hobby team was added. You can check your username at https://vercel.com/account

If you're seeing warnings about a username already being in use, that's most likely because it really is already in use on another account. But if there does seem to be a problem with your username please let me know the current and intended usernames so I can look into it for you.

**Comment by dogangok** [comment]
Hi Amy, thanks for the quick response. My initial username on GitHub was 'skyborn', which I also used for deployments within Vercel. I changed my GitHub username from 'skyborn' to 'dogangok' and assigned 'skyborn' as an organization on GitHub: https://github.com/skyborn. Since your recent automatic update, 'skyborn' was automatically changed to 'dogangok'. I'm convinced that 'skyborn' is still linked to my accounts in the system and thus marked as in use, but I cannot access it. I would greatly appreciate it if you could change my account back from 'dogangok' to 'skyborn’.

**Reply by amyegan** [reply, marked as answer]
Thanks for letting me know the usernames. For security reasons I am not able to change your username. However, I was able to look into the username history for your current account.

I do not see any history of the username `skyborn` existing on the `dogangok` account. Do you have a second account that may have been previously created with your preferred username? If that's the case then you should be able to log in with the associated email address to access that account.

**Comment by dogangok** [comment]
Thank you for your quick response, time, and patience, Amy. Due to the
migration, I have used several different names and email addresses,
including skyborn, skyborn2, skyborn-archive, ***@***.***, and
***@***.***

Could you check which email address or name is associated with the Vercel
account under the username 'skyborn'? I believe something might have gone
wrong there during the migration.

Thanks in advance.

**Reply by amyegan** [reply]
For security reasons, I cannot provide the full email address but I can give you an obscured email that you may recognize: `s****r***@g****.c**`

**Comment by dogangok** [comment]
Hi Amy, thank you for your quick response.

I do not recognize the obscured email address. It seems that I lost the
Vercel username 'skyborn' during the unannounced migration, even though I
still own the GitHub username connected to 'github.com/skyborn'.

I understand the need for the migration, but better communication during
this process would have been greatly appreciated to avoid such issues. I
hope there is a way to resolve this and reclaim the 'skyborn' username on
Vercel.

Thank you for your assistance.

Best regards

[a comment by an automated account is not shown]
